src/controller/python/chip/commissioning/pase.py

```diff
@@ -0,0 +1,55 @@
+#
+#    Copyright (c) 2023 Project CHIP Authors
+#    All rights reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+import dataclasses
+
+from chip import ChipDeviceCtrl
+from chip import commissioning
+
+
+@dataclasses.dataclass
+class Session:
+    node_id: int
+    device: ChipDeviceCtrl.DeviceProxyWrapper
+
+
+class ContextManager:
+    def __init__(self, devCtrl: ChipDeviceCtrl.ChipDeviceControllerBase, node_id: int, is_ble: bool):
+        self.devCtrl = devCtrl
+        self.node_id = node_id
+        self.is_ble = is_ble
+
+    def __enter__(self) -> Session:
+        return Session(
+            node_id=self.node_id,
+            device=self.devCtrl.GetConnectedDeviceSync(self.node_id, True, 1000))
```

Comment:
Why is the 1000 by defualt? Could you make the arguments explicit?

Reply:
Changed to using keyword arguments.